oracle/blockoracle.go

```diff
@@ -107,7 +107,8 @@ func (bo *MinerBlockOracle) calcEligibilityProofs(epochNumber types.EpochId) err
 		})
 	}
 	bo.proofsEpoch = epochNumber
-	bo.log.Info("miner \"%v…\" is eligible for blocks on %d layers in epoch %d", bo.nodeID.Key[:5], len(bo.eligibilityProofs), epochNumber)
+	bo.log.Info("miner %v is eligible for %d blocks on %d layers in epoch %d",
```

Comment:
Any reason not to prefer %v?
Using %v will not require you to change the print code if you change the param type.

Reply:
no special reason, but also -- why would I change the param type?

Reply:
Why would a programmer change his/someone else's code? (;

Reply:
Let me clarify: `%d` works for all number types. This variable is a number and it doesn't make sense to change it to a non-number type. If that would happen, this log would need to change anyway and the IDE would show a warning on this line.

Reply:
I have no objection to using `%v`, but the IDE puts `%d` there when I do `option`+`return` and select `Add format string argument` and I see no reason to change that.